examples/minimal/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@examples/minimal",
+  "private": true,
+  "version": "0.1.0",
+  "scripts": {
+    "dev:server": "nodemon -e ts -w . -x ts-node ./server/index.ts",
+    "dev:client": "nodemon -e ts -w . -x 'wait-port 2022 && ts-node ./client/index.ts'",
+    "dev": "run-p dev:* --print-label",
+    "start": "node server",
+    "build": "tsc",
+    "test-dev": "start-server-and-test 'ts-node server' 2022 'ts-node client'",
+    "test-start": "start-server-and-test 'node server' 2022 'node client'"
+  },
+  "workspaces": [
+    "client",
+    "server"
+  ],
+  "devDependencies": {
+    "@types/node": "^17.0.10",
+    "@types/node-fetch": "^2.5.11",
```

Comment:
Shouldn't this only be a dependency of the `client` workspace?

Reply:
fixed